subiquity/common/types/storage.py

```diff
@@ -420,3 +420,8 @@ class EntropyResponse:
 
     # Set to None if success is True
     failure_reasons: Optional[List[str]] = None
+
+
+class CoreBootTPMKeyEncryption(enum.Enum):
```

Comment:
suggestion: `CoreBootTPMMethod` or `CoreBootTPMEncryptionMethod` if the former is too vague.

If you were to take the above suggestion then we could talk about encryption "methods" that we "support". So the GET request would give us a list of supported methods:
```python
class core_boot_tpm_support:
                def GET() -> List[CoreBootTPMMethod]: ...
```

If you wanted to mirror the snapd type we could use "features" instead of "methods" too.